pkg/receipt/receipt_test.go

```diff
@@ -68,3 +70,20 @@ func TestStore(t *testing.T) {
 		t.Fatal(diff)
 	}
 }
+
+func TestLoad(t *testing.T) {
+	p, err := Load(filepath.Join("..", "..", "integration_test", "testdata", "foo.yaml"))
```

Comment:
This doesn't feel right. Depending on test data in another package looks like headache in the future.
What about writing the plugin manifest to disk and read with `Load`? For that it could be helpful to have a `testutil.ValidPluginManifest` global value which can be used by other tests as well.

Reply:
> What about writing the plugin manifest to disk and read with `Load`?

this is a good idea.

> For that it could be helpful to have a `testutil.ValidPluginManifest` global value

this is mentioned in #270

Reply:
Hmm let's keep it as is, I'll add a TODO here. I have two concerns:

1. I don't want to do #270 in this PR.
2. Ideally I don't want Load()’s tests to fail because of a fault in Store(). But I think we can live with it.